data/_ui/tooltips.txt

```diff
@@ -667,10 +667,10 @@ tip "shield protection:"
 	`Protection against shield damage. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in shield damage, while a total value of 11 only results in a 10 percent reduction.`
 
 tip "silent jumps:"
-	`Prevents a ship from making hyperdrive or jump drive sounds when entering or leaving a system.`
+	`Prevents a ship from emitting sounds when entering or leaving a system via hyperjump.`
```

Comment:
```suggestion
	`Prevents a ship from making hyperdrive or jump drive sounds when entering or leaving a system.`
```
I believe the original tooltip is correct because it specifically mentions what makes the sounds. I also don't think "emits" is a "fix" to the hyperdrive sound line, if we're keeping the original intent; it's just a different way of saying it, so this should stay as-is.